superdesk/publish/formatters/__init__.py

```diff
@@ -112,6 +112,23 @@ def map_html_to_xml(self, element, html):
         if len(list(element)) == 0:
             SubElement(element, 'p').text = soup.get_text()
 
+    def pass_through(self, article):
```

Comment:
It will be used by a number of formatter implimentations :- IPNews, Newscentre, ANPA and Bulletin builder. All AAP specific. I could move it to the AAP repo if you like!

The Medianet, Racing system, pagemasters and AP ingests will be set to pass through.